DER/DerivativesContracts/Forwards.rdf

```diff
@@ -260,113 +434,169 @@
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardContract"/>
-		<rdfs:range rdf:resource="&fibo-der-der-fwd;ForwardValuationTermsSet"/>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;Forward"/>
+		<rdfs:range rdf:resource="&fibo-der-der-fwd;ForwardValuationTerms"/>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;hasValuationTimeOfDay">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDateTime"/>
 		<rdfs:label xml:lang="en">has valuation time of day</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;MasterConfirmationValuationTimeTermsSet"/>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ValuationTiming"/>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;DateTime"/>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;hasValuationTiming">
 		<rdfs:label xml:lang="en">has valuation timing</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardValuationTermsSet"/>
-		<rdfs:range>
-			<owl:Class>
-				<owl:unionOf rdf:parseType="Collection">
-					<rdf:Description rdf:about="&fibo-der-der-fwd;MasterConfirmationValuationTimeTermsSet">
-					</rdf:Description>
-					<rdf:Description rdf:about="&fibo-der-der-fwd;ValuationTimeType">
-					</rdf:Description>
-					<rdf:Description rdf:about="&fibo-fnd-dt-fd;DateTime">
-					</rdf:Description>
-				</owl:unionOf>
-			</owl:Class>
-		</rdfs:range>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardValuationTerms"/>
+		<rdfs:range rdf:resource="&fibo-der-der-fwd;ValuationTiming"/>
 	</owl:ObjectProperty>
 	
-	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;paymentFeeType">
-		<rdfs:label xml:lang="en">payment fee type</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-der-der-fwd;ForwardCashSettlementCommitment"/>
-		<rdfs:range rdf:resource="&xsd;string"/>
-		<skos:definition xml:lang="en">The type of fee or additional payment, e.g. brokerage, upfront fee etc.</skos:definition>
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;lastDeliveryDate">
+		<rdfs:label xml:lang="en">last delivery date</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Future"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
+		<skos:definition xml:lang="en">The last date in the range of dates by which the underlying for a futures contract must be delivered in order for the terms of the contract to be fulfilled.</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-der-der-fwd;lastNoticeDate">
+		<rdfs:label xml:lang="en">last notice date</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Future"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
+		<skos:definition xml:lang="en">The last date at which a delivery notice can be issued.</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-der-der-fwd;marginCall">
+		<rdfs:label xml:lang="en">has margin call</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-der-der-fwd;FuturesListing"/>
```

Comment:
This seems utterly wrong. It's not on the listing but the specific contract surely. And the margin call is a specific event on a specific date and for a specific amount of money.

I now see this is a boolean. Does it refer to a listed option potentially being subject to a margin call? In which case it's still wrong - the margin relates to a specific purchase by a specific buyer not the listing. Someone might buy the same listing without using margin.
In fact the margin call will be on an account not a specific purchase.